src/main/scala/nodes/learning/LBFGS.scala

```diff
@@ -183,8 +208,13 @@ class SparseLBFGSwithL2(
     val numCorrections: Int = 10,
     val convergenceTol: Double = 1e-4,
     val numIterations: Int = 100,
-    val regParam: Double = 0.0)
-  extends LabelEstimator[SparseVector[Double], DenseVector[Double], DenseVector[Double]] {
+    val regParam: Double = 0.0,
+    val sparseOverhead: Double = 8)
```

Comment:
Can you document what `sparseOverhead` is ? (Also I'm genuinely curious about what it is ?)